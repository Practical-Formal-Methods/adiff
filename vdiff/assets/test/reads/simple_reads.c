
void foo(int x) {
}

void bar();

int main() {

  int x,y,z;

  if (x > 0) {
    int a;
    foo(3);
    y = y + 1;
    bar();
    x = x + 1;
    z = z + 1;
  }

}
