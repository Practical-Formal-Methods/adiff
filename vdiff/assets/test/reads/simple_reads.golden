void foo();
void bar();
int main()
{
    int x, y, z;
    __VERIFIER_read(x);
    if (x > 0)
    {
        int a;
        foo();
        __VERIFIER_read(y);
        y = y + 1;
        bar();
        __VERIFIER_read(x);
        x = x + 1;
        __VERIFIER_read(z);
        z = z + 1;
    }
}